event_pubsub/event_repository.py

```diff
@@ -122,10 +122,9 @@ def insert_registry_event(self, block_number, event_name, json_str, processed, t
 
             insert_query = "INSERT INTO registry_events_raw (block_no, event, json_str, processed, transactionHash, logIndex ,error_code,error_msg,row_updated,row_created) " \
                            "VALUES ( %s, %s, %s, %s, %s , %s, %s, %s, %s, %s ) " \
-                           "ON DUPLICATE KEY UPDATE event=%s, json_str=%s, processed=%s, logIndex=%s , error_code=%s, error_msg=%s, row_updated=%s "
+                           "ON DUPLICATE KEY UPDATE row_updated=%s "
```

Comment:
Lets unify all these methods as only the table names seem to be changing

Reply:
as discussed will create an issue and try to pick up asap.

Reply:
will create the issue for this and pick based on priority